helm/reana/values.yaml

```diff
@@ -124,7 +124,7 @@ ingress:
   enabled: true
   annotations:
     kubernetes.io/ingress.class: traefik
-    traefik.frontend.entryPoints: "http,https"
+    traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
     ingress.kubernetes.io/ssl-redirect: "true"
```

Comment:
Is `ingress.kubernetes.io/ssl-redirect` still needed with traefik v2?

Reply:
Not sure about this one, I also wanted to remove it as it seems it's used for traefik 1.7, but after testing it on DEV it didn't work without it. It could be that I have tested not the final version of the new ingress resource, so we could re-test it again with custom Docker image on DEV